liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/InstallFeatureMojo.java

```diff
@@ -90,4 +97,11 @@ private void installFeaturesFromAnt(List<Feature> installFeatures) {
         installFeatureTask.execute();
     }
 
+    public boolean isOpenLibertyBetaVersion(String openLibertyVersion) {
+        if (openLibertyVersion != null && openLibertyVersion.endsWith("-beta")) {
+            return true;
+        }
+        return false;
+    }
+
```

Comment:
Consider moving this into InstallFeatureUtil (e.g. as a static method) as well, so that it can also be used in ci.gradle